yarn.lock

```diff
@@ -4848,7 +4848,7 @@ __metadata:
   resolution: "@serlo/editor-web-component@workspace:packages/editor-web-component"
   dependencies:
     "@rollup/plugin-replace": ^5.0.5
-    "@serlo/editor": 0.7.0
+    "@serlo/editor": 0.7.2
```

Comment:
In the editor package.json, it changes to 0.7.3, but here it changes to 0.7.2, is that expected?